Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Build Stage
```

Comment:
Please do not use git clone or `ldd` in your Dockerfile. You don't need to set the AFL env vars for Mayhem (`AFL_SKIP_CPUFREQ` and `AFL_I_DONT_CARE_ABOUT_MISSING_CRASHES`) as we will handle this.